syntax/gha.vim

```diff
@@ -8,16 +8,18 @@ set cpo&vim
 
 let s:gha_keywords_key = '\%(^\|\s\)\@<=\zs\%('.join(gha#GetKeywords(), '\|').'\)\ze\%(\s*:\|$\)'
 let s:gha_keywords_conditional_key = '\%(^\|\s\)\@<=\zs\%('.join(gha#GetKeywordsConditional(), '\|').'\)\ze\%(\s*:\|$\)'
-let s:gha_keywords_step_key = '\%([0-9A-Za-z_-]\)\@<!\%('.join(gha#GetKeywordsStep(), '\|').'\)\ze\%(\s*:\|$\)'
+let s:gha_keywords_step_key = '\%([0-9A-Za-z_-]\)\@<!\%>6c\%('.join(gha#GetKeywordsStep(), '\|').'\)\ze\%(\s*:\|$\)'
```

Comment:
Since there are some common keywords between workflow and steps (typically `name`), I added this constraint that step keywords should be found _at least_ after the 6th column.

Reply:
I misunderstood the difference between [\%>l](https://vim-jp.org/vimdoc-ja/pattern.html#/\%%3El) and [\%>c](https://vim-jp.org/vimdoc-ja/pattern.html#/\%%3Ec). Thanks.